File: HumanitarianAssistance.WebApi/NewUI/src/app/dashboard/accounting/chart-of-accounts/models/chart-of-account.model.ts

export class ChartOfAccountModel {
    ChartOfAccountNewId?: number;
    AccountName?: string;
    ChartOfAccountNewCode?: string;
    ParentID?: number;
    AccountLevelId?: number;
    AccountHeadTypeId: number;
    AccountTypeId?: number;
    AccountFilterTypeId?: number;
    Children?: ChartOfAccountModel[] = [];
    _IsDeleted?: boolean;
    _IsLoading?: boolean;
    _IsError?: boolean;
}

export class AccountFilterTypeModel {
    AccountFilterTypeId: number;
    AccountFilterTypeName: string;
}

export class AccountTypeModel {
    AccountTypeId: number;
    AccountTypeName: string;
    AccountCategory: number;
    AccountHeadTypeId: number;
}

File: HumanitarianAssistance.Application/Accounting/Models/ChartOfAccountModel.cs

namespace HumanitarianAssistance.Application.Accounting.Models
{
    public class ChartOfAccountModel
    {
        public long ChartOfAccountId { get; set; }
        public string ChartOfAccountCode { get; set; }
        public string AccountName { get; set; }
        public long ParentID { get; set; }
        public int AccountLevelId { get; set; }
    }
}